src/index.html

```diff
@@ -1,13 +1,55 @@
 <!doctype html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
         content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <title>Product cards</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Product cards</h1>
-</body>
+    <title>Product cards</title>
+    <link rel="stylesheet" href="./style.css">
+  </head>
+  <body>
+    <div class="productcard" data-qa="card">
+      <div class="images">
+        <img
+          class="images_item"
+          src="images/imac.jpeg"
+         alt="foto"
+     >
+      </div>
+      <div class="description">
```

Comment:
1. correct attributes position
2. space between siblings

fix it throughout your code
```suggestion
    <div class="productcard" data-qa="card">
      <div class="images">
        <img
           class="images_item"
           src="images/imac.jpeg"
           alt="foto"
        >
      </div>
      
      <div class="description">
```

Reply:
is not fixed. it should be 
`    <div class="productcard" data-qa="card">
      <div class="images">
        <img
          class="images_item"
          src="images/imac.jpeg"
          alt="foto"
        >
      </div>`